zilencer/management/commands/populate_db.py

```diff
@@ -774,6 +775,19 @@ def assign_time_zone_by_delivery_email(delivery_email: str, new_time_zone: str)
                     {"id": pronouns.id, "value": "he/him"},
                 ],
             )
+            # We need to create at least one scheduled message for Iago for the api-test
+            # cURL example to delete an existing scheduled message.
+            check_schedule_message(
+                sender=iago,
+                client=get_client("populate_db"),
+                recipient_type_name="stream",
+                message_to=[Stream.objects.get(name="Denmark", realm=zulip_realm).id],
+                topic_name="test-api",
+                message_content="For cURL example delete scheduled message",
```

Comment:
This shows up in the dev environment, so we probably want to reword this to be more in keeping with the messages in that context. I was thinking this would only be in tests when I wrote it.

Reply:
Yeah, I think something like "Time to celebrate the anniversary of provisioning the development environment :tada:!" would be pretty reasonable.